youtube_dl/extractor/adn.py

```diff
@@ -129,19 +132,31 @@ def _get_subtitles(self, sub_url, video_id):
             }])
         return subtitles
 
+    def _real_initialize(self):
+        username, password = self._get_login_info()
+        if username:
+            accessToken = (self._download_json(
+                self._API_BASE_URL + "authentication/login", None,
+                'Logging in', data=urlencode_postdata({
+                    'password': password,
+                    'rememberMe': False,
+                    'source': "Web",
+                    'username': username,
+                }), fatal=False, errnote="Unable to log in") or {}).get('accessToken')
```

Comment:
- use single quotes consistently.
- i think it would be better to keep errnote closer to note.